### ISSUE TITLE
Review Registration and Reporting header

### ISSUE THREAD

**Issue by emileighoutlaw** [18F/fec-cms#283]
My critique group was kind enough to look at our registration and reporting landing page today, and they had fantastic comments on content and design. 

<img width="1021" alt="screen shot 2016-04-22 at 12 01 53 pm" src="https://cloud.githubusercontent.com/assets/10552702/14747518/9902cc48-0882-11e6-8429-b60dc3603747.png">

#### Registration and reporting subhead
**Current subhead:**
> Register your candidacy or committee and get started reporting

This sounds like folks can register and report on these pages. Really they just contain information **about** registering and reporting. It's also not inclusive of researchers and journalists.

**Proposed new subhead options:**
> Information about candidates, committees and volunteers

or

> Learn more about the process and requirements

I think I like the latter option better. @jenniferthibault, @noahmanger, @amykort and @amypike, what do you think? 

#### View registration categories
If my memory serves, we included this arrow down button when the designs had a lot more content for `What does it mean to register and report?` My critique colleagues were surprised the button scrolled only a small bit down the page. They were also confused about what `registration categories` meant. 

Which got me thinking, once we trim the content per https://github.com/18F/fec-cms/issues/282, there will be even less text in this section. 

Do we really need this button + text, @jenniferthibault, and @noahmanger? 

#### Already registered?
If folks use this search box, it takes them to the data section. I can't remember why we added this here. My gut thinks it'd be fine to not have it at all (and expect folks to go back to the top nav to get to the data pages).

But if we have strong reasons why we want it, we should probably clarify that it searches the data profile pages, perhaps mirroring our new navigation language:

> **Already registered?**
> Search for candidates and committees in any election

**Comment by AmyKort** [comment]
Couple of thoughts: For the Already Registered box, I think this was an opportunity for a committee that was already registered to look up their own reports. If I remember correctly, there was an expressed user need to have an easy way to find a record of one's own committee's filed reports on the FEC's website. As the page has evolved, though, I don't think this is a natural place for an existing committee to look for a way to locate the public record of their files. As it is, some of our internal users have also found the box confusing.

For the new subhead: I agree that we should change the one we have, but I'm not 100% excited about the two options. Helpfully, I also cannot think of any text myself. 

Is it something like "Learn more about when and how to register and report"? Too long? Too wordy?

**Comment by jenniferthibault** [comment]
**View registration categories / jump arrow**
Seeing the content shortened in #282 and desire to shorten the hero space overall in #330 I'm 👍 with eliminating these.

**Already registered**
Amy hit on the user need here, a fast way for existing committees to find their information on record, in the absence of having user accounts or logins. It's appearance on this page is to redirect those users right off the bat, but it also appears on internal Essentials pages as well, to give people a chance to redirect there (https://beta.fec.gov/registration-and-reporting/essentials-house-and-senate-candidates-and-committees/ ) This feature's appearance also promotes cross-site traveling outside of the navigation, and more in context of the page. We _may_ be able to get analytics to see if it's used, but I'm not sure. 
(Our existing usability test format wouldn't be a good format to see if people use this or not, because they aren't structured as observations. Analytics or observations would be better.)

**Comment by emileighoutlaw** [comment]
Thanks for the notes, Amy! And thanks of the **Already registered?** context, Jen!

#### Search
I'd be curious to know if we _can_ get analytics on that box, but I can an argument for cross-site traveling (outside of the navigation). ( @noahmanger, do you want to weigh in on this? ) 

If we keep the box, what do y'all think of this rewording:

> **Already registered?**
> Search for candidates and committees in any election


#### Subheads
I'm happy to think about some other options for this @amykort! I'd love to know more about what _doesn't_ excite you about the first two options. 

Mainly, what I'm trying to avoid here is saying "register and report" directly below "Registration and reporting." If folks need more context to understand what "Registration and reporting" means, we should probably try to say something other than "register and report" directly below?

**Comment by AmyKort** [comment]
I thought the other options were too broad--I wasn't sure what information or process we were talking about. How about "Understand your filing requirements" or "Learn about federal filing requirements" or something like that?

**Comment by emileighoutlaw** [comment]
I see what you're getting at! How about:

`Learn about requirements for candidates, committees, and volunteers`

**Comment by AmyKort** [comment]
I like it. I wonder if we should hold off on the volunteers bit until we see that content. Since the point is that volunteers don't have registration and reporting, I'm afraid that might be misleading if we put it up front. Also, do you think it is worth adding "federal" right up there, since that seems to be a frequent point of confusion for first-time committees? Or do you think we've already covered that enough?

**Comment by emileighoutlaw** [comment]
Good thought! So we'd say:

`Requirements for federal candidates and committees` 

And then we could add "volunteers" later. I love it, if you do!

**Comment by AmyKort** [comment]
I love it, too. Thank you for thinking through all of this!
